locales/sv-SE/src/structure.ts

```diff
@@ -80,6 +80,8 @@ export default removeUndefinedLocaleResources({
   'banners.deleted-document-banner.restore-button.text': 'Återställ senaste versionen',
   /** The text content for the deleted document banner */
   'banners.deleted-document-banner.text': 'Detta dokument har raderats.',
+  /** The text content for the deprecated document type banner */
+  'banners.deprecated-document-type-banner.text': 'Denna dokumenttyp har avvecklats.',
```

Comment:
Not entirely sure but maybe better with `'Denna dokumenttyp har inaktiverats'`?

Reply:
Personally I think your second suggestion sounds the best of the three.

Reply:
I'll vote for `Denna dokumenttyp är inte längre tillgänglig.`

Reply:
+1

Reply:
```suggestion
  'banners.deprecated-document-type-banner.text': 'Denna dokumenttyp är inte längre tillgänglig.',
```